hail_search/queries/mito.py

```diff
@@ -354,9 +508,10 @@ def _add_project_lookup_data(self, ht, annotation_fields, *args, **kwargs):
         if not variant_projects:
             raise HTTPNotFound()
 
+        self._has_both_sample_types = True
```

Comment:
is this always true?

Reply:
This is set to True because it forces `_get_sample_genotype` to return a list of familyGenotypes instead of getting the genotype for the first sample. 

A few commits ago I had an additional flag `all_samples`  but decided to use _has_both_sample_types instead